core/src/components/range/test/basic/range.e2e.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from '@playwright/test';
+import { dragElementBy, test } from '@utils/test/playwright';
+
+test.describe('range: basic', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/range/test/basic`);
+
+    await page.setIonViewport();
+
+    expect(await page.screenshot({ fullPage: true })).toMatchSnapshot(`range-diff-${page.getSnapshotSettings()}.png`);
```

Comment:
If you do `setIonViewport`, do we even need to do `fullPage: true`?

Reply:
same question goes for other instances of this in the PR

Reply:
Yep, good catch! Github Copilot auto completed those sections. They aren't necessary when the test template uses `ion-content` + `setIonViewport()`. I will remove 👍